samples/echoServer/build.gradle

```diff
@@ -17,3 +17,28 @@ kotlin {
 MPPTools.createRunTask(project, 'runProgram', kotlin.targets.echoServer) {
     args 3000
 }
+
+private def determinePreset() {
```

Comment:
We shall build for both host and few cross targets, like it's done in `samples/androidNativeActivity`

Reply:
ok. I will rework.